tests/test_date.nit

```diff
@@ -0,0 +1,13 @@
+import date
```

Comment:
It does not please me but I agree with @Morriar. Tests of libs should be done in nitunits or testsuites. Cf #1035